app/analytics/routes.py

```diff
@@ -160,15 +160,16 @@ def pipelines_views():
                     v.nb_uniq_visitors if v.nb_uniq_visitors is not None else 0)
 
         if not exists and v.label is not None and "/pipeline?id=" in v.label:
-            element = {
-                "url": v.url,
-                "label": v.label,
-                "title": pipelines.get_title_from_id(v.label.split('id=')[1]),
-                "nb_hits": v.nb_hits,
-                "nb_visits": v.nb_visits,
-                "nb_uniq_visitors": v.nb_uniq_visitors if v.nb_uniq_visitors is not None else 0,
-            }
-            elements.append(element)
+            if pipelines.get_title_from_id(v.label.split('id=')[1]):
```

Comment:
You are assuming that `id` is the only query in the url, what if there are multiple queries like `/pipeline?id=123&arg=456` or `/pipeline?arg=123&id=456&arg2=789`. Would this scenario be possible?

Reply:
@joeyzhou98 I just looked into the database just in case. The URLs gathered by Matomo and present in the database table are always of the form:
`https://portal.conp.ca/pipeline?id=zenodo.1234567`
There is no other argument in the URL of a tool on the portal.